pybikes/whitebikes.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) Eduardo Mucelli Rezende Oliveira <edumucelli@gmail.com>
+# Distributed under the AGPL license, see LICENSE.txt
+
+import json
+
+from pybikes.base import BikeShareSystem, BikeShareStation
+from pybikes import utils
+
+
+class WhiteBikes(BikeShareSystem):
+
+    meta = {
+        'system': 'WhiteBikes'
+    }
+
+    def __init__(self, tag, meta, feed_url, bounding_box=None):
+        super(WhiteBikes, self).__init__(tag, meta)
+        self.feed_url = feed_url
+
+    def update(self, scraper=None):
+        scraper = scraper or utils.PyBikesScraper()
+
+        stations = []
+
+        data = json.loads(scraper.request(self.feed_url))
+
+        for station in data:
+
+            longitude = float(station['lon'])
+            latitude = float(station['lat'])
+
+            name = station['standName']
+            free = 0
+            bikes = int(station['bikecount'])
+
+            extra = {
+                'photo': station['standPhoto'],
+                'description': station['standDescription'],
+                'number': int(station['standId'])
+            }
+
+            station = BikeShareStation(name, latitude, longitude, bikes, free,
+                                       extra)
+            stations.append(station)
+
+        self.stations = list(stations)
```

Comment:
There should be no need to explicitly create a copy of `stations`, `self.stations = stations` should work.

Reply:
Yep